### PR TITLE
fix: colors in console output from worker

### LINKED ISSUE #15366: [REGRESSION]: Console log object values are not highlighted
**Context:**
- GOOD Playwright Version: 1.22.2
- BAD Playwright Version: 1.23.1
- Operating System: Windows 10, 21H2
- PowerShell: 7.2.5
- Windows Terminal: 1.13.11431.0

**Snippet**
```
test('Object highlight test', () => {
    console.log({ myBoolean: true, myNumber: 123, myString: 'abc' });
});
```

**Description**
- In Playwright v1.22.2, console log object string values are highlighted green, and booleans and numbers are yellow:
image

- In Playwright v1.23.1, the font highlighting is gone :-(
image

### PR DESCRIPTION
**Pull request by yury-s**
Color highlights regressed in https://github.com/microsoft/playwright/pull/14455

Fixes #15366

### CODE REVIEW

File: packages/playwright-test/src/worker.ts [hunk 1]
```diff
@@ -24,6 +24,9 @@ let closed = false;
 
 sendMessageToParent('ready');
 
+if (process.env.FORCE_COLOR === '1')
+  process.stdout.isTTY = true;
```
[the remark this answers is not in the document]
Comment (yury-s) [reply packages/playwright-test/src/worker.ts:28]: Done.

**Review by mxschmitt — APPROVED** [review]
nice fix, also looked into it but didn't came to such a solution.

File: tests/playwright-test/stdio.spec.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import * as fs from 'fs';
```
Comment (mxschmitt) [inline tests/playwright-test/stdio.spec.ts:17]:
```suggestion
```

File: packages/playwright-test/src/worker.ts [hunk 3]
```diff
@@ -24,10 +24,17 @@ let closed = false;
 
 sendMessageToParent('ready');
 
-if (process.env.FORCE_COLOR === '1') {
-  process.stdout.isTTY = true;
-  process.stderr.isTTY = true;
-}
+// Make sure the output supports colors.
+process.stdout.isTTY = true;
+if (process.env.PWTEST_STDOUT_ROWS)
```
[the remark this answers is not in the document]
Comment (yury-s) [reply packages/playwright-test/src/worker.ts]: Moved this to WorkerInitParams instead, it still runs before any user code so should be early enough and cleaner.

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**